love/src/components/FlightTracker/FlightTracker.jsx

```diff
@@ -0,0 +1,252 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import MapFlightTracker from './MapFlightTracker';
+import Value from '../GeneralPurpose/SummaryPanel/Value';
+import Title from '../GeneralPurpose/SummaryPanel/Title';
+import StatusText from '../GeneralPurpose/StatusText/StatusText';
+import Button from '../GeneralPurpose/Button/Button';
+import isEqual from 'lodash';
+import SimpleTable from 'components/GeneralPurpose/SimpleTable/SimpleTable';
+import ManagerInterface from 'Utils';
+import ZoomInIcon from 'components/icons/Zoom/ZoomInIcon';
+import ZoomOutIcon from 'components/icons/Zoom/ZoomOutIcon';
+import moment from 'moment';
+import { flightTrackerStateToMap, flightTrackerStatetoStyle } from 'Config';
+import styles from './FlightTracker.module.css';
+
+const DEFAULT_POLLING_TIMEOUT = 5000;
+
+export default class FlightTracker extends Component {
+  static propTypes = {
+    status: PropTypes.number,
+    aircrafts: PropTypes.arrayOf(PropTypes.objectOf({
+      id: PropTypes.string,
+      latitude: PropTypes.number,
+      longitude: PropTypes.number,
+      altitude: PropTypes.number,
+      track: PropTypes.number,
+      distance: PropTypes.number,
+      speed: PropTypes.number,
+    })),
+
+  };
+
+  static defaultProps = {
+    status: 0,
+    aircrafts: [],
+  };
+
+  constructor(props) {
+    super(props);
+    this.pollingInterval = null;
+    this.countPollingIterval = null;
+    this.state = {
+      lastUpdate: 0,
+      zoom: '200',
+      aircraftInRadius: 0,
+    };
+    this.RADIO1 = 160; // Distance of Warning
+    this.RADIO2 = 100; // Distance of Alert
```

Comment:
I would set these as constants at the beginning of the file, similar to line 17 (to be potentially moved to the Config or Constants file).